src/main/java/pl/magisterka/dominikszojda/controller/DogOwnerController.java

```diff
@@ -0,0 +1,27 @@
+package pl.magisterka.dominikszojda.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import pl.magisterka.dominikszojda.entity.DogOwnerEntity;
+import pl.magisterka.dominikszojda.request.DogOwnerCreateRequestDto;
+import pl.magisterka.dominikszojda.response.DogOwnerResponseDto;
+import pl.magisterka.dominikszojda.service.DogOwnerService;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("dog-owners")
+public class DogOwnerController {
+
+    private final DogOwnerService dogOwnerService;
+
+    @PostMapping("")
+    public DogOwnerResponseDto createDogOwner(@RequestBody DogOwnerCreateRequestDto dto) {
+        return dogOwnerService.createDog(dto);
+    }
+    @GetMapping("")
+    public List<DogOwnerEntity> getDogOwners() {
+        return dogOwnerService.getDogOwners();
+    }
```

Comment:
W innych controllerach używasz `ResponseEntity` jako return value, dobrze dla spójności zrobić to samo, plus dodatkowo, ResponseEntity daje ci możliwość kontrolowania innych rzeczy z response, tak jak np. kodu odpowiedzi (dla operacji create, zwykło się dawać `201 CREATED` tak jak masz w `DogController#createDog`)